osquery/database/plugins/rocksdb.cpp

```diff
@@ -218,7 +220,7 @@ Status RocksDBDatabasePlugin::setUp() {
   }
 
   // RocksDB may not create/append a directory with acceptable permissions.
-  if (!read_only_ && chmod(path_.c_str(), S_IRWXU) != 0) {
```

Comment:
Not sure how I feel about using the literal `0700` instead of `S_IRWXU`. I understand it isn't available on Windows but perhaps we can `#define` within `fileops.h` if the expected define from (`sys/types.h`?) does not exist?